rasa/utils/common.py

```diff
@@ -53,41 +53,6 @@ def read_global_config() -> Dict[Text, Any]:
         return {}
 
 
-def write_global_config_value(name: Text, value: Any) -> None:
```

Comment:
both functions are actually used by `rasa-x` and indirectly used e.g. in `check_license_and_metrics`

Reply:
I think we should keep it in `rasa` btw. There might be some other information we want to store in their in the future (e.g. some default parameters)

Reply:
good to know. will add them again.